lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -255,6 +257,12 @@ func (handler *ContainersHandlersImpl) RemoveContainerHandler(params containers.
 		case exec.RemovePowerError:
 			return containers.NewContainerRemoveConflict().WithPayload(&models.Error{Message: err.Error()})
 		default:
+			if f, ok := err.(types.HasFault); ok {
+				switch f.Fault().(type) {
+				case *types.HostNotConnected:
+					return containers.NewContainerRemoveDefault(500).WithPayload(&models.Error{Message: "Couldn't remove container. The ESX host is temporarily disconnected. Please try again later."})
```

Comment:
Suggest replacing 500 with the constant from the `http` pkg. With that, I imagine this line will be quite lengthy, so we can declare the error message string in a separate line.

Reply:
oh right yeah I saw your other comment and forgot :+1: